client/src/BuyerTenderList.js

```diff
@@ -81,7 +81,11 @@ const BuyerTenderList = () => {
 					<tbody>
 						{buyerTenders.map((tender) => (
 							<tr key={tender.id}>
-								<td>{tender.id}</td>
+								<td>
+									<a className="tender-id" href={`./bidding/${tender.id}`}>
+										{tender.id}
+									</a>
```

Comment:
Is react-router able to handle this correctly?

Reply:
I have tested it. its working.